specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/examples/VirtualNetworkCreateSubnetWithDelegation.json

```diff
@@ -0,0 +1,113 @@
+{
+  "parameters" : {
+    "api-version": "2018-08-01",
+    "subscriptionId" : "subId",
+    "resourceGroupName" : "rg1",
+    "virtualNetworkName" : "test-vnet",
+    "parameters": {
+      "properties": {
+        "addressSpace": {
+          "addressPrefixes": [
+            "10.0.0.0/16"
+          ]
+        },
+        "subnets": [
+          {
+            "name": "test-1",
+            "properties": {
+              "addressPrefix": "10.0.0.0/24",
+              "delegations": [
+                {

```

Comment:
Same here - `ServiceDelegationPropertiesFormat` doesn't contain name, properties. You probably wanted to return a list of ServiceDelegation here.

Reply:
Fixed, thanks!